docs/playwright.md

```diff
@@ -0,0 +1,243 @@
+---
+permalink: /playwright
+title: Testing with Playwright
+---
+
+# Testing with Playwright
+
+Playwright is a Node library to automate the [Chromium](https://www.chromium.org/Home), [WebKit](https://webkit.org/) and [Firefox](https://www.mozilla.org/en-US/firefox/new/) browsers with a single API. It enables **cross-browser** web automation that is **ever-green**, **capable**, **reliable** and **fast**.
+
+The primary goal with Playwright is to improve automated UI testing by eliminating flakiness, improving the speed of execution and offering insights into the browser operation.
+
+CodeceptJS uses Playwright to improve end to end testing experience. No need to learn the syntax of a new tool, all drivers in CodeceptJS share the same API.
+
+Take a look at a sample test:
+
+```js
+I.amOnPage('https://github.com');
+I.click('Sign in', '//html/body/div[1]/header');
+I.see('Sign in to GitHub', 'h1');
+I.fillField('Username or email address', 'something@totest.com');
+I.fillField('Password', '123456');
+I.click('Sign in');
+I.see('Incorrect username or password.', '.flash-error');
+```
+
+It's readable and simple and working using Playwright API!
+
+## Setup
+
+To start you need CodeceptJS with Playwright packages installed
+
+```bash
+npm install codeceptjs playwright --save
+```
+
+Or see [alternative installation options](http://codecept.io/installation/)
+
+> If you already have CodeceptJS project, just install `playwright` package and enable a helper it in config.
+
+And a basic project initialized
+
+```sh
+npx codeceptjs init
+```
+
+You will be asked for a Helper to use, you should select Playwright and provide url of a website you are testing.
+
+> Puppeteer can also work with Firefox, Webkit
+
+|          | ver | Linux | macOS | Win |
+|   ---:   | :---: | :---: | :---:  | :---: |
+| Chromium| 81.0.4044 | :white_check_mark: | :white_check_mark: | :white_check_mark: |
+| WebKit | 13.0.4 | :white_check_mark: | :white_check_mark: | :white_check_mark: |
+| Firefox |73.0b3 | :white_check_mark: | :white_check_mark: | :white_check_mark: |
+- Headless is supported for all the browsers on all platforms.
+
+## Configuring
+
+Make sure `Playwright` helper is enabled in `codecept.conf.js` config:
+
+```js
+{ // ..
+  helpers: {
+    Playwright: {
+      url: "http://localhost",
+      show: true
```

Comment:
playwright requires browser option

Reply:
I saw you put the default option there https://github.com/Codeception/CodeceptJS/pull/2173/files#diff-810cf1fbb95328f07ac7468f9a4dd270R182